README.md

```diff
@@ -45,7 +45,7 @@ on AWS.
 * For a detailed walkthrough, see [Getting Started] in the AWS CDK [User Guide]
 * See [Manual Installation](./MANUAL_INSTALLATION.md) for installing the CDK from a signed .zip file
 
-Install the [AWS CDK Toolkit] from npm (requires [Node.js ≥ 8.11.x](https://nodejs.org/en/download)):
+Install the [AWS CDK Toolkit] or update your installed version of the |cdk| to the latest version from npm (requires [Node.js ≥ 8.11.x](https://nodejs.org/en/download)):
```

Comment:
Can we just phrase as:

```
Install or update the [AWS CDK Toolkit] using npm:
```

And be done with it? The shorter the easier to read.

Reply:
Sure